airflow/contrib/hooks/aws_hook.py

```diff
@@ -40,10 +40,14 @@ class AwsHook(BaseHook):
     Interact with AWS.
     This class is a thin wrapper around the boto3 python library.
 
-    :param str aws_conn_id: The Airflow connection used for AWS credentials.
-       If this is None then the default boto3 behaviour is used. If running Airflow
-       in a distributed manner and aws_conn_id is None, then default boto3 configuration
-       would be used (and must be maintained on each worker node).
+    :param aws_conn_id: The Airflow connection used for AWS credentials.
+        If this is None then the default boto3 behaviour is used. If running Airflow
+        in a distributed manner and aws_conn_id is None, then default boto3 configuration
+        would be used (and must be maintained on each worker node).
+    :type aws_conn_id: str
+    :param verify: Whether or not to verify SSL certificates.
+        https://boto3.amazonaws.com/v1/documentation/api/latest/reference/core/session.html
+    :type verify: str or bool
```

Comment:
Hm, this type looks strange 🤔

Reply:
I am open for suggestions :)
boto3 docs: verify (boolean/string)
it could be `Union[str, bool]`, also PyCharm quick documentation (ctrl+q) renders `str or bool` as `Union[str, bool]`
![image](https://user-images.githubusercontent.com/18196182/73398937-68279000-42f7-11ea-9d5f-50e2508d23f4.png)